<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>770</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pbOpenData">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Open TS</string>
         </property>
         <property name="icon">
          <iconset resource="Pics.qrc">
           <normaloff>:/new/Icons/pics/OpenFolder.png</normaloff>:/new/Icons/pics/OpenFolder.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbCloseData">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Close TS</string>
         </property>
         <property name="icon">
          <iconset resource="Pics.qrc">
           <normaloff>:/new/Icons/pics/CloseFolder.png</normaloff>:/new/Icons/pics/CloseFolder.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbSave">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
         <property name="icon">
          <iconset resource="Pics.qrc">
           <normaloff>:/new/Icons/pics/Save.png</normaloff>:/new/Icons/pics/Save.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Num. Harmonics</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spNumHarmonics">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbRawTransferFunction">
            <property name="text">
             <string>Raw TRF</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbComputeTF">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Compute TF</string>
         </property>
         <property name="icon">
          <iconset resource="Pics.qrc">
           <normaloff>:/new/Icons/pics/Button_StartNow.png</normaloff>:/new/Icons/pics/Button_StartNow.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbShowPlot">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Show Plot</string>
         </property>
         <property name="icon">
          <iconset resource="Pics.qrc">
           <normaloff>:/new/Icons/pics/ShowPlot.png</normaloff>:/new/Icons/pics/ShowPlot.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTableWidget" name="tableTSFiles">
         <column>
          <property name="text">
           <string>Use for TF</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Sample Freq. [Hz]</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Calibration Freq. [Hz]</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Sensor Ex</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Sensor Ey</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Sensor Hx</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Sensor Hy</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Sensor Hz</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Target Directory</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1092</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Pics.qrc"/>
 </resources>
 <connections/>
</ui>
